src/components/SearchForm/SearchForm.jsx

```diff
@@ -1,15 +1,15 @@
 import React, { useState } from 'react';
 import './SearchForm.css';
 
-export default function SearchForm({ onSearchSumbit }) {
+export default function SearchForm({ onSearchSubmit }) {
   const [keyword, setKeyword] = useState('');
 
-  const handleSumbit = (e) => {
+  const handleSubmit = (e) => {
     e.preventDefault();
-    onSearchSumbit(keyword);
+    onSearchSubmit(keyword);
   };
   return (
-    <form onSubmit={handleSumbit} className="search-form">
+    <form onSubmit={handleSubmit} className="search-form">
       <input onChange={(event) => setKeyword(event.target.value)} maxLength="50" className="input input__search" type="text" placeholder="Enter topic" />
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components